_posts/2017-08-27-Clean-Architecture.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+author: kris
+header-img: "img/architecture-bg.jpg"
+disqus: true
+---
+# Application Architecture: let's sort things out
+
+## Lego's
+If you're reading this, and you're someone with a technical background chances are quite high you've build something using lego's during your lifetime. You might have just gotten new boxes. That's usually quite easy. You get the pieces in nice plastic bags and it comes with a manual. If you've ever gotten lego's from family members or friends that have already been played with, well, not so easy. You get big boxes of mixed together parts, nice creations destroyed to their back to their basics parts without any resemblance or clue what they ever were.
+
+This is where you can let your imagination run wild and start building whatever you like, no manual, just scrounging through the parts and picking anything that suits you.
+
+![Random built item](/img/lego-simple.jpg)
+
+## Professional lego building
+Now imagine if you got into this on a professional level. And you have these huge boxes of unsorted lego's. And they come to you with the question of building something quite large.
+
+![Lego cathedral](/img/cathedral.jpg)
+
+If you would need to start building a lego cathedral you'd probably want to spent a bit more time investing in structure. Structure in the way you work. Consistency in how you build certain parts. Making sure you don't loose time finding where pieces are. And making sure that when you need more help you can easily explain to people how they can help you and what they should do.
```

Comment:
ik zou hier discipline ok vermelden. een gestructureerde aanpak die men volgt die men toelaat om snel te blijven gaan. iedereen weet waar alles ligt during build time. etc

De metaphoor houdt wel niet eenmaal het gebouwd is. zal je je cathedraal nog veel wijzigen eenmaal gebouwd? Nieuwe features toevoegen?